src/components/organisms/Accordion/AccordionHeader/AccordionHeader.tsx

```diff
@@ -44,8 +45,15 @@ const AccordionHeader: FC<IAccordionHeader> = ({ children, id, index, textSize =
   const { getHeaderProps, isActiveSection } = useAccordionContext();
   const { ref, onClick: contextOnClick, ...headerPropsRest } = getHeaderProps(id, index);
 
+  const willBecomeActive = !isActiveSection(index);
```

Comment:
I think `willBecomeActive` is also a bit confusing since it's essentialy `inactive`. I would just stick to `isActive`

Reply:
There's no easy way around the confusion here.

Reply:
I personally would leave `isActive` in here and then you can rename it while implementing it in the app

Reply:
I updated the code and readme